packages/react/src/components/Theme/index.js

```diff
@@ -9,8 +9,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import { usePrefix } from '../../internal/usePrefix';
+import { LayerContext } from '../Layer/LayerContext';
```

Comment:
Same thing here with the path needing updated to include `next`. Or maybe `LayerContext` should be pulled out of the `next` folder?

Reply:
Good call, updated!